profiles/kentik_snmp/a10_networks/a10-thunder.yml

```diff
@@ -153,6 +154,9 @@ metrics:
       # The count of free SMP connections.
       - name: axSessionGlobalStatConnSMPFree
         OID: 1.3.6.1.4.1.22610.2.4.3.19.1.14.0
+      # OSPF Status 
+      - name: axOSPFstatus
+        OID: 1.3.6.1.2.1.14.10.1.6
```

Comment:
thanks for you submission; a couple of points:

1. highly recommend leaving the metric name as the original name (`ospfNbrState`) to ensure you don't end up with 2X metric collection in case OSPF monitoring is added as a default later in this repo.
2. The OID should be suffixed with `.0` as a scalar OID to match the patterns we use in these profiles.
3. This metric [requires enumeration to decorate the actual state names](https://www.circitor.fr/Mibs/Html/O/OSPF-MIB.php#ospfNbrState)
  * example in this profile: `axVirtualServerStatDisplayStatus`